Apps/phvmray/vmray_connector.py

```diff
@@ -1,119 +1,136 @@
-                            VMRAY_ERR_SERVER_CONNECTION,
-                            VMRAY_ERR_SUBMISSION_NOT_FINISHED,
-                            VMRAY_ERR_SUBMIT_FILE,
-                            VMRAY_ERR_UNSUPPORTED_HASH,
-                            VMRAY_JSON_API_KEY,
-                            VMRAY_JSON_DISABLE_CERT,
-                            VMRAY_JSON_SERVER,
-                            VMRAY_SEVERITY_BLACKLISTED,
-                            VMRAY_SEVERITY_ERROR,
-                            VMRAY_SEVERITY_MALICIOUS,
-                            VMRAY_SEVERITY_NOT_SUSPICIOUS,
-                            VMRAY_SEVERITY_SUSPICIOUS,
-                            # VMRAY_SEVERITY_WHITELISTED,
-                            # VMRAY_SEVERITY_UNKNOWN,
-                            VMRAY_SUCC_CONNECTIVITY_TEST,
-                         )
-
-from rest_cmds import VMRay  # pylint: disable=relative-import
-
-if False:  # pylint: disable=using-constant-test
-    # pylint: disable=unused-import,import-error
-    from typing import Any, Text, Tuple, Union  # NOQA
-
-# pylint: disable=broad-except
-
-
-def _analysis_severity_by_score(score):
-    # type: (int) -> Text
-    """Return severity for given analysis score"""
-
-    if score < 25:
-        return VMRAY_SEVERITY_NOT_SUSPICIOUS
-    elif score < 75:
-        return VMRAY_SEVERITY_SUSPICIOUS
-    else:
-        return VMRAY_SEVERITY_MALICIOUS
-
-
-# see sample_model.py severity(self, user)
-def _severity(highest_vti_score, reputation_severity):
-    # type: (int, Text) -> Text
-
-    if highest_vti_score is not None:
-        # there exists at least one analysis score
-        analyses_severity = _analysis_severity_by_score(highest_vti_score)
-
-        if analyses_severity == VMRAY_SEVERITY_MALICIOUS:
-            # analysis score says malicious -> return malicious
-            return VMRAY_SEVERITY_MALICIOUS
-        elif reputation_severity == VMRAY_SEVERITY_BLACKLISTED:
-            # reputation says blacklisted -> return blacklisted
-            return VMRAY_SEVERITY_BLACKLISTED
-        elif (analyses_severity == VMRAY_SEVERITY_SUSPICIOUS) or (
-                reputation_severity == VMRAY_SEVERITY_SUSPICIOUS):
-            # analysis score or reputation say suspicious -> return
-            # suspicious
-            return VMRAY_SEVERITY_SUSPICIOUS
-        else:
-            # analysis score says not suspicious, reputation says
-            # something uninteresting -> return not suspicious
-            return VMRAY_SEVERITY_NOT_SUSPICIOUS
-    elif (reputation_severity is not None) and (
-            reputation_severity != VMRAY_SEVERITY_ERROR):
-        # no analysis score exists, reputation severity exists  -> return
-        # mapped reputation result
-        return reputation_severity
-    else:
-        # no analysis score or reputation severity exists -> return None
-        return None
+from typing import Any, Dict, List, Tuple, Union
+
+# pylint: disable=E0401
```

Comment:
let's use the pylint error names instead of codes for readability (same for all other uses of codes across all files)

Reply:
Agree. Updated the connector accordingly.